test/bucket.ts

```diff
@@ -44,10 +44,12 @@ function FakeFile(bucket, name, options?) {
   this.createWriteStream = options => {
     self.metadata = options.metadata;
     const ws = new stream.Writable();
-    (ws as any).write = () => {
-      ws.emit('complete');
-      ws.end();
-    };
+    extend(ws, {
```

Comment:
This doesn't feel right.  Instead of this, could we create an interface that has the `write` method and use that?

Reply:
That's a good point. `WriteStream#write()` is actually a thing, we just need to replace `util.noop` to a function that returns a `boolean`. Addressed in bd2911b